ext/setup/theme.php

```diff
@@ -99,4 +99,56 @@ protected function sb_to_html(SetupBlock $block): string
 		";
         return $html;
     }
+
+    protected function build_setup_chunk(string $html): string
+    {
+        return "<table class='form'>$html</table>";
+    }
+
+    protected function build_setup_row(string $html): string
+    {
+        return "<tr>$html</tr>";
+    }
+
+    protected function build_setup_cell(string $html, bool $is_header=false, bool $full_width=false): string
+    {
+        $colspan = ($is_header || $full_width ? 2 : 1);
+        if ($is_header) {
+            return $this->build_setup_row("<th colspan='$colspan'>$html</th>");
+        } else {
+            if ($full_width) {
+                return $this->build_setup_row("<td colspan='$colspan'>$html</td>");
+            } else {
+                return "<td colspan='$colspan'>$html</td>";
+            }
+        }
+    }
+
+    public function format_item(?string $label, ?string $html, ?string $config_name, bool $label_is_header=false, bool $full_width=false): string
+    {
+        if (empty($label) && empty($html)) {
+            return "";
+        }
+
+        if (!empty($label)) {
+            if (!empty($config_name)) {
+                $label = "<label for='{$config_name}'>{$label}</label>";
+            }
+            $label = $this->build_setup_cell($label, $label_is_header, $full_width);
+        } else {
+            $label = "";
+        }
+
+        if (!empty($html)) {
+            $html = $this->build_setup_cell($html, $label_is_header, $full_width);
+        } else {
+            $html = "";
+        }
+
+        if ($label_is_header) {
+            return $label . $this->build_setup_row($html);
+        }
+        
+        return $this->build_setup_row($label . $html);
```

Comment:
Found a bug here. I'll work on it tomorrow.
It's pretty harmless tho, creates an empty table row when formatting a full width label.